snapcraft.yaml

```diff
@@ -103,12 +103,9 @@ plugs:
     interface: system-files
     read: [/etc/firefox]
   host-hunspell:
-    interface: mount-control
-    mount:
-    - what: /usr/share/hunspell
-      where: $SNAP_COMMON/host-hunspell
-      persistent: true
-      options: [ro, bind, noatime, noexec]
+    interface: system-files
```

Comment:
This will require an update to the firefox snap declaration in the store.

Reply:
Yes. Do you know if the store can handle the case where the plug name is unchanged? Maybe it'd be better if I changed the name to say `host-usr-share-hunspell`?

Reply:
I've renamed the plug to host-usr-share-hunspell to avoid potential name conflicts in snap declaration.

Reply:
Thanks, yep I think this is better.

Reply:
Submitted store request for review/approval:

[Request for auto-connection of firefox:host-usr-share-hunspell plug of system-files interface](https://forum.snapcraft.io/t/request-for-auto-connection-of-firefox-host-usr-share-hunspell-plug-of-system-files-interface/39619)

Reply:
FTR, the store request was granted, so this thread can be resolved.